examples/client/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 project (test_client LANGUAGES CXX)
 
+find_package(yaml-cpp "0.7.0" REQUIRED)
```

Comment:
Isn't there a way of reusing the same version string in all of the cmakelists?

Reply:
yes, like in boost, will fix